tools/download.sh

```diff
@@ -71,23 +71,51 @@ case "$OSTYPE" in
     linux*)
         source /etc/os-release
         # Install dependencies
+        echo "Installing dependencies with sudo..."
         case "$ID" in
             centos)
-                echo "Installing libicu, libunwind, and $package with sudo ..."
-                sudo yum install -y libicu libunwind
+                sudo yum install -y \
+                     glibc \
+                     libcurl \
+                     libgcc \
+                     libicu \
+                     openssl \
+                     libstdc++ \
+                     ncurses-base \
+                     libunwind \
+                     uuid \
+                     zlib
                 sudo yum install "./$package"
                 ;;
             ubuntu)
                 case "$VERSION_ID" in
                     14.04)
```

Comment:
Eh this is installing literally the packages that will be listed as dependencies by the package.